views/userPage.html

```diff
@@ -127,7 +127,7 @@
                   }
 
               // Inject the whole content string into our existing HTML table
-              $('#bandList').html(content);
+
```

Comment:
are we changing how this injection is done? will accept review & you can explain me later :)